### DISCUSSION TITLE
Update additional fields in an explicit many-to-many relationship

### DISCUSSION THREAD

**Discussion by binajmen** [prisma/prisma#11465] [Q&A] [answered]
Hi 👋 

With the following schema:
```prisma
model Exam {
  id          String         @id @default(uuid()) @db.Uuid
  createdAt   DateTime       @default(now())
  updatedAt   DateTime       @updatedAt
  name        String
  description String
  chapters    ExamChapters[]
}

model ExamChapters {
  exam      Exam    @relation(fields: [examId], references: [id])
  examId    String  @db.Uuid // relation scalar field (used in the `@relation` attribute above)
  chapter   Chapter @relation(fields: [chapterId], references: [id])
  chapterId String  @db.Uuid // relation scalar field (used in the `@relation` attribute above)
  order     Int

  @@id([examId, chapterId])
}

model Chapter {
  id          String             @id @default(uuid()) @db.Uuid
  createdAt   DateTime           @default(now())
  updatedAt   DateTime           @updatedAt
  name        String
  description String
  exams       ExamChapters[]
}
```

After a reordering, I would like to update the order value in the m-to-n relationship `ExamChapters`:
```ts
    order.map(async (id, index) => {
      await prisma.examChapters.update({
        where: { ... },
        data: { order: index },
      });
    });
```

In the `where`, I was expecting to be able to provide `examId: "...", chapterId: id` but instead intellisense suggest `examId_chapterId`.

How am I suppose to use examId_chapterId? By concatenating the two IDs?

**Comment by binajmen** [comment, marked as answer]
```ts
    order.map(async (id, index) => {
      await prisma.examChapters.update({
        where: { examId_chapterId: { examId: params.examId as string, chapterId: id } },
        data: { order: index },
      });
    });
```
I suppose this is documented somewhere but I was not able to get my hand on

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
